src/utils/index.js

```diff
@@ -40,3 +42,25 @@ export function dividers (num: number): Array<number> {
   }
   return divs
 }
+
+export function getNoDefaultConsoleErrorMessages (reason: string, vmName: string, hasRdp: boolean, defaultConsole: string): string {
```

Comment:
Why do we need  `hasRdp` for distinguishing between non rdp and console on current solution?  Ithink we should have one message referring to console only.